frontend/src/scenes/data-warehouse/external/dataWarehouseJoinsLogic.ts

```diff
@@ -46,7 +46,7 @@ export const dataWarehouseJoinsLogic = kea<dataWarehouseJoinsLogicType>([
                     if (table) {
                         acc.push(
                             ...Object.values(table.fields).map((column) => ({
-                                id: `${join.field_name}.${column.name}`,
+                                id: `${join.field_name}: ${column.name}`,
```

Comment:
We split by `.` elsewhere too - so I'd recommend keeping it as a dot separator and change the `property_to_expr`

Reply:
https://github.com/PostHog/posthog/pull/22685/files#diff-9e284b4a4f022ffe99ed273e872c3f351daf79c73b4964860abf92300bed185fR32

Reply:
https://github.com/PostHog/posthog/pull/22685/files#diff-66a83f26c784f7721e4df1189a3cf662dd5dac9319aa15a60c0a8ad781d84ccfR191

Reply:
(or change the above 2 files to be colons, but I do think a dot is likely better)

Reply:
changed to dot! and updated test